include/powsybl/iidm/PhaseTapChangerHolder.hpp

```diff
@@ -22,9 +22,13 @@ class PhaseTapChangerHolder : public virtual TapChangerHolder {
 public:
     ~PhaseTapChangerHolder() noexcept override = default;
 
-    virtual stdcxx::CReference<PhaseTapChanger> getPhaseTapChanger() const = 0;
+    virtual stdcxx::CReference<PhaseTapChanger> getOptionalPhaseTapChanger() const;
 
-    virtual stdcxx::Reference<PhaseTapChanger> getPhaseTapChanger() = 0;
+    virtual stdcxx::Reference<PhaseTapChanger> getOptionalPhaseTapChanger();
+
+    virtual const PhaseTapChanger& getPhaseTapChanger() const = 0;
+
+    virtual PhaseTapChanger& getPhaseTapChanger() = 0;
```

Comment:
Why hasPhaseTapChanger is not declared in this interface?

Reply:
It is inherited from `TapChangerHolder` and still virtual pure in `RatioTapChangerHolder`/`PhaseTapChangerHolder`. Do I have to redeclare it in both subclasses `RatioTapChangerHolder`/`PhaseTapChangerHolder` ?